dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/partitions/KafkaPartitionTest.java

```diff
@@ -56,14 +62,20 @@ public static void setup() throws Exception {
         adminClientProps.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, PlaintextTests.kafkaContainer.getBootstrapServers());
         AdminClient adminClient = AdminClient.create(adminClientProps);
 
-        NewTopic newTopic = new NewTopic(PartitionTestReceptionBean.CHANNEL_NAME, 2, (short) 1);
-        adminClient.createTopics(Collections.singleton(newTopic)).all().get(KafkaTestConstants.DEFAULT_KAFKA_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+        List<NewTopic> newTopics = new ArrayList<>();
+        newTopics.add(new NewTopic(PartitionTestReceptionBean.CHANNEL_NAME, 2, (short) 1));
```

Comment:
is 2 partitions enough to properly exercise this code?

Reply:
I think so. We're really just looking to test the case where a partition is assigned, removed, then assigned again to test that messages are handled correctly. We could add more partitions though we might then need to process more messages and make the test take longer and I'm already a bit concerned about how long this test will take.